### PR TITLE
0.2.4 release notes

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR adds release notes for 0.2.4 and updates our release guide to indicate our new approach using release candidates on pypi

### CODE REVIEW

File: docs/release/release_0_2_4.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Improvements
+------------
+- Make _on_data_change only get called once when ndisplay changes (#629)
+- fix undefined variables, remove unused imports (#633)
+- Raise informative warning when no Qt loop is detected (#642)
+
+API Changes
+-----------
+
```
Comment (jni) [inline docs/release/release_0_2_4.rst:26]: `napari.Viewer.add_multichannel` was removed. Use `napari.Viewer.add_image(..., channel_axis=num)` instead.

Comment (jni) [reply docs/release/release_0_2_4.rst:26]: Oh yeah, this is an API change, not a deprecation, which means, something is now discouraged and will be removed in a future version.

**Review by jni — APPROVED** [review]
I've left a comment, otherwise approved!

File: docs/release/release_0_2_4.rst (hunk 1 above)
Comment (sofroniewn) [reply docs/release/release_0_2_4.rst:26]: changed

**Comment by jni** [comment]
Cool, feel free to merge when tests pass (though mostly a formality for this kind of change

**Pull request merged**